go/common/syscall/syscall_darwin.go

```diff
@@ -0,0 +1,9 @@
+// Package syscall defines OS-specific syscall parameters.
+package syscall
+
+import "syscall"
+
+const IoctlTermiosGetAttr = syscall.TIOCGETA
+
+// CmdAttrs is the SysProcAttr that will ensure graceful cleanup (on Linux).
```

Comment:
Maybe add a comment saying that `PR_SET_PDEATH_SIG` is not implemented on Darwin and so this is empty and as a consequence the plugin subprocesses may not always be cleaned up.